### PR TITLE
Add - Redux 

### PR DESCRIPTION
**Pull request by joshualyngdoh**
# Description

- Add redux in components
- Ran linters
- Update README

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
   return (
     <div>
       <ul>
-        <li>{title}</li>
-        <li>{author}</li>
+        <h3>{title}</h3>
+        <h4>{author}</h4>
       </ul>
-      <button type="button">Remove</button>
+      <button type="button" id={id} onClick={removeHandler}>Remove</button>
```
Comment (OLIPLICHE) [inline src/components/Book.js]: - [x] I would advise you to implement removing books; adding and removing books must be possible.  It is the way to confirm if the code you implemented is working.
- 
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @joshualyngdoh, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 1 above)
Comment (joshualyngdoh) [reply src/components/Book.js]: Thank you for the feedback

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @joshualyngdoh 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.
- Your linters are passing all the checks.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joshualyngdoh** [comment]
Thank you for the feedback

**Pull request merged**